GoodFirstIssue-ToDoApp/Mode.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  Mode.swift
+//  GoodFirstIssue-ToDoApp
+//
+//  Created by Yoshiki Izumi on 2020/09/21.
+//  Copyright © 2020 GoodFirstIssue. All rights reserved.
+//
+
+import Foundation
+
+enum Mode {
+    case create
+    case edit(id: Int, name: String)
```

Comment:
ここは趣旨を踏まえると
```suggestion
    case edit(item: Item)
```
でもよいような？

Reply:
ありがとうございます。 edit(id: Int, name: String) から edit(item: Item) に変更しました。